packages/salesforcedx-utils-vscode/package.json

```diff
@@ -11,7 +11,7 @@
   "main": "out/src",
   "dependencies": {
     "@salesforce/core-bundle": "^8.10.3",
-    "@salesforce/salesforcedx-utils": "64.1.0",
+    "@salesforce/salesforcedx-utils": "*",
```

Comment:
merge conflicts are annoying.  `*` for monorepos means, the one in the monorepo.

so it should be a lot more resistant to git merge issues.